core/cmd/shell_local.go

```diff
@@ -223,6 +223,29 @@ func initLocalSubCmds(s *Shell, safe bool) []cli.Command {
 						},
 					},
 				},
+				{
+					Name:    "delete-chain",
+					Aliases: []string{},
+					Usage:   "Commands for cleaning up chain specific db tables. WARNING: This will ERASE ALL chain specific data referred to by --chainType and --id options for the specified database, referred to by CL_DATABASE_URL env variable or by the Database.URL field in a secrets TOML config.",
+					Action:  s.CleanupChainTables,
+					Before:  s.validateDB,
+					Flags: []cli.Flag{
+						cli.IntFlag{
+							Name:     "id",
+							Usage:    "chain id based on which chain specific table cleanup will be done",
+							Required: true,
+						},
+						cli.StringFlag{
+							Name:     "chainType",
```

Comment:
WDYT about calling this one just `type`, since we already know `chain` from context?
```suggestion
							Name:     "type",
```

Reply:
Both are fine with me. I usually prefer full contex with naming, but I guess context won't be an issue here so let's go with just type to follow the id flag naming